pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --cov ./deepparse --cov-report html --cov-report xml --cov-config=.coveragerc
+#addopts = --cov ./deepparse --cov-report html --cov-report xml --cov-config=.coveragerc
```

Comment:
?

Reply:
Forgot to remove, I need to comment this line to enable debugging while testing. Will revert the changes